mantidproject/mantid
# New 3D-deltaPDF algorithm

This add a new algorithm DeltaPDF3D that will calculate the 3D-deltaPDF from a HKL MDHistoWorkspace

**To test:**
Here are some example MDHisto you can use for testing: `/SNS/users/rwp/corelli/IPTS-16617-PMN/PMN_normdata_48sym_300K.nxs`, `/SNS/users/rwp/corelli/IPTS-18149-Dy2Ti2O7/DTO_normdata_48sym_680mK_H0.8T.nxs`, `/SNS/users/rwp/benzil/benzil_300K_bkg_subtract_sym_All_noCC_flat2.nxs`

Run the DeltaPDF3D algorithm on these (read the docs) and see that something comes out the other side.


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
